Source/Core/Core/HW/DSP.cpp

```diff
@@ -374,33 +374,30 @@ void RegisterMMIO(MMIO::Mapping* mmio, u32 base)
                  }));
 
   // Audio DMA MMIO controlling the DMA start.
-  mmio->Register(
-      base | AUDIO_DMA_CONTROL_LEN, MMIO::DirectRead<u16>(&s_audioDMA.AudioDMAControl.Hex),
-      MMIO::ComplexWrite<u16>([](u32, u16 val) {
-        bool already_enabled = s_audioDMA.AudioDMAControl.Enable;
-        s_audioDMA.AudioDMAControl.Hex = val;
-
-        // Only load new values if were not already doing a DMA transfer,
-        // otherwise just let the new values be autoloaded in when the
-        // current transfer ends.
-        if (!already_enabled && s_audioDMA.AudioDMAControl.Enable)
-        {
-          s_audioDMA.current_source_address = s_audioDMA.SourceAddress;
-          s_audioDMA.remaining_blocks_count = s_audioDMA.AudioDMAControl.NumBlocks;
+  mmio->Register(base | AUDIO_DMA_CONTROL_LEN,
+                 MMIO::DirectRead<u16>(&s_audioDMA.AudioDMAControl.Hex),
+                 MMIO::ComplexWrite<u16>([](u32, u16 val) {
+                   bool already_enabled = s_audioDMA.AudioDMAControl.Enable;
+                   s_audioDMA.AudioDMAControl.Hex = val;
 
-          INFO_LOG_FMT(AUDIO_INTERFACE, "Audio DMA configured: {} blocks from {:#010x}",
-                       s_audioDMA.AudioDMAControl.NumBlocks, s_audioDMA.SourceAddress);
+                   // Only load new values if were not already doing a DMA transfer,
+                   // otherwise just let the new values be autoloaded in when the
+                   // current transfer ends.
```

Comment:
Just noticed a typo here; this should use `we're`.  (This existed beforehand, but it's easy enough to fix it now)